### PR TITLE
fix: server issues

### PR DESCRIPTION
**Pull request by jsmassa**
1. Copies data folder to container to enable category mapping.
2. Removes references to 'site' directory which had been removed from the container in a prior commit.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -12,8 +12,7 @@ FROM node:21
 
 COPY --from=build /heisse-preise-de/server/build /heisse-preise-de/server/build
 COPY --from=build /heisse-preise-de/package.json /heisse-preise-de/package.json
-
-EXPOSE 3000
```
Comment (TimoKramer) [inline Dockerfile:16]: is this not correct?

File: package.json [hunk 2]
```diff
@@ -5,9 +5,9 @@
   "main": "server.js",
   "scripts": {
     "format": "npx prettier --write .",
-    "clean": "rimraf site/build && rimraf server/build",
+    "clean": "rimraf server/build",
     "build:server": "node ./esbuild.server.mjs",
-    "dev": "npm run build:server && npm run server",
+    "dev": "npm run build:server && PGUSER=prices PGHOST=localhost PGPASSWORD=prices PGDATABASE=prices PGPORT=5433 npm run server",
```
Comment (TimoKramer) [inline package.json:10]: I don't know if it is a good idea to have the config here, but fine for me

**Review by TimoKramer — APPROVED** [review]
Thanks for fixing this and sorry for the delay. Will deploy asap after merged.

File: package.json (hunk 2 above)
Comment (jsmassa) [reply package.json:10]: It's just for development and it's convenient, so I don't see a reason why not

File: Dockerfile (hunk 1 above)
Comment (jsmassa) [reply Dockerfile:16]: We can leave it, but the web site is gone, so there is no reason to expose it anymore. Or am I overlooking something?

File: package.json (hunk 2 above)
Comment (TimoKramer) [reply package.json:10]: sure you're right

File: Dockerfile (hunk 1 above)
Comment (TimoKramer) [reply Dockerfile:16]: sure np

**Pull request merged**